src/Utils/Helpers.ts

```diff
@@ -65,29 +65,33 @@ export class RealPath {
   }
 }
 
+export function getErrorMessage(error: unknown) {
+  if (error instanceof Error) {
+    return error.message;
+  }
+  return String(error);
```

Comment:
Q) Does this work for every type of instances?

Reply:
I just copy-pasted the line, so I will check. Thanks for pointing it out.

Reply:
https://stackoverflow.com/questions/5612787/converting-an-object-to-a-string

It seems that the javascript String(); function converts any object into string.

Or, `JSON.stringify(yourobject)` looks more explicit, so I can replace it with this.

Reply:
As the link explains, `String(obj)` might not give sufficient information for user. How about replacing it to `JSON.stringify`? It can be done in another PR :smile: